### DISCUSSION TITLE
ship version w/o all connectors by default

### DISCUSSION THREAD

**Discussion by o-az** [wevm/wagmi#2459] [Ideas]
Not sure if this is possible yet. Can I use wagmi without all the connectors included in the install?
If not then would you consider shipping a version that doesn't include all the connectors by default? `@coinbase/wallet-sdk` and `safe-global/safe-apps-sdk` are quite heavy and I've never used them in any project.

**Comment by jxom** [comment]
If you don't import those connectors, then they won't be included in your final bundle.
